core/Cargo.toml

```diff
@@ -39,32 +39,23 @@ default-members = ["."]
 members = [".", "fuzz", "edge/*", "benches/vs_*"]
 
 [features]
-default = [
-  "rustls",
-  "services-azblob",
-  "services-azdls",
-  "services-cos",
-  "services-fs",
-  "services-gcs",
-  "services-ghac",
-  "services-http",
-  "services-ipmfs",
-  "services-memory",
-  "services-obs",
-  "services-oss",
-  "services-s3",
-  "services-webdav",
-  "services-webhdfs",
-  "services-azfile",
-]
+default = ["rustls"]
 
 # Build test utils or not.
 #
 # These features are used to control whether to build opendal's test utils.
 # And doesn't have any other effects.
 #
 # You should never enable this feature unless you are developing opendal.
-tests = ["dep:rand", "dep:sha2", "dep:dotenvy", "layers-blocking"]
+tests = [
+  "dep:rand",
+  "dep:sha2",
+  "dep:dotenvy",
+  "layers-blocking",
+  "services-azblob",
+  "services-memory",
+  "services-http",
```

Comment:
This affects a little dev experience. Now `cargo test` without `--features tests` cannot compile.